tests/e2e/vcp_to_csi_syncer.go

```diff
@@ -957,11 +959,15 @@ var _ = ginkgo.Describe("[csi-vcp-mig] VCP to CSI migration syncer tests", func(
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		err = fdep.WaitForDeploymentComplete(client, dep1)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		pods, err = fdep.GetPodsForDeployment(client, dep1)
+		framework.Logf("sleep for 1 min...")
+		time.Sleep(1 * time.Minute)
```

Comment:
is this sleep required?

Reply:
Yes, other wise, it will fail with,

```
  Unexpected error:
      <*errors.errorString | 0xc0001c4590>: {
          s: "expected a new replica set for deployment \"deployment-be3470bc-1bd4-4167-81f3-a70e0de4e606\", found none",
      }
      expected a new replica set for deployment "deployment-be3470bc-1bd4-4167-81f3-a70e0de4e606", found none
  occurred
```